modules/react/button/lib/DeleteButton.tsx

```diff
@@ -1,75 +1,54 @@
 import * as React from 'react';
-import {ButtonColors} from './types';
-import {BaseButton, BaseButtonProps, getMinWidthStyles, getPaddingStyles} from './BaseButton';
-import {
-  GrowthBehavior,
-  useTheme,
-  Themeable,
-  EmotionCanvasTheme,
-  createComponent,
-} from '@workday/canvas-kit-react/common';
 
-export interface DeleteButtonProps extends Themeable, GrowthBehavior, BaseButtonProps {
-  /**
-   * The size of the Button.
-   */
-  size?: 'small' | 'medium' | 'large';
-  children?: React.ReactNode;
-}
+import {buttonVars, ButtonContainerProps} from './BaseButton';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {base, brand, system} from '@workday/canvas-tokens-web';
+import {Button} from './Button';
 
-const getDeleteButtonColors = ({
-  canvas: {
-    palette: {error: themeError},
-  },
-}: EmotionCanvasTheme): ButtonColors => ({
-  default: {
-    background: themeError.main,
-    label: themeError.contrast,
-  },
-  hover: {
-    background: themeError.dark,
+/**
+ * Extends all the style properties from Box to our buttons as well as props from ButtonContainerProps.
+ * We omit `ref` since all of our buttons use `createComponent` and already give access to `ref`.
+ * Use this type to extend and customize any one off buttons that you want full control over styling.
+ */
+export interface DeleteButtonProps extends Omit<ButtonContainerProps, 'ref'> {}
```

Comment:
Ahh. Yeah I left that in since it's an exportable interface and if anyone was using that, I didn't want it to cause an issue if I removed it. That goes for all the other buttons as well.

Reply:
Ahh I see what you mean